be/src/olap/rowset/segment_v2/column_zone_map.h

```diff
@@ -17,8 +17,9 @@
 
```

Comment:
rename this file name to zone_map_index.h

Reply:
OK